openhtf/io/proto/guzzle.proto

```diff
@@ -0,0 +1,33 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
It's to match the internal format, which has it as a separate file. True, it's a tiny file, but it's along a logical line (format vs upload path), merging them is a huge pain, and combining them is a small gain.